src/epost/PrescriptionEpostService.tsx

```diff
@@ -0,0 +1,371 @@
+            value={formData.postCode}
+            margin="normal"
+            required
+            fullWidth
+            onChange={(e) =>
+              setFormData({ ...formData, postCode: e.target.value })
+            }
+          />
+        </div>
+        <div>
+          <Typography className={styles.subHeader} variant="h6">
+            Sender details
+          </Typography>
+          <Typography variant="subtitle1" className={styles.italicText}>
+            The address of the patient who got the prescription
+          </Typography>
+          <TextField
+            id="senderAddressLineOne"
+            name="senderAddressOne"
+            label={'Sender Address Line 1'}
+            value={formData.senderAddressLineOne}
+            margin="normal"
+            required
+            fullWidth
+            onChange={(e) =>
+              setFormData({
+                ...formData,
+                senderAddressLineOne: e.target.value,
+              })
+            }
+          />
+          <TextField
+            id="addressTwo"
+            name="addressTwo"
+            label={'Address Line 2'}
+            value={formData.senderAddressLineTwo}
+            margin="normal"
+            required
+            fullWidth
+            onChange={(e) =>
+              setFormData({
+                ...formData,
+                senderAddressLineTwo: e.target.value,
+              })
+            }
+          />
+          <TextField
+            id="city"
+            name="city"
+            label={'City'}
+            value={formData.senderCity}
+            margin="normal"
+            required
+            fullWidth
+            onChange={(e) =>
+              setFormData({ ...formData, senderCity: e.target.value })
+            }
+          />
+          <TextField
+            id="postcode"
+            name="postcode"
+            label={'Postcode'}
+            value={formData.senderPostCode}
+            margin="normal"
+            required
+            fullWidth
+            onChange={(e) =>
+              setFormData({ ...formData, senderPostCode: e.target.value })
+            }
+          />
+        </div>
+        <div>
+          <Typography className={styles.subHeader} variant="h6">
+            PDF Upload
+          </Typography>
+          <div className={styles.uploadBox}>
+            <Typography component="h4" variant="body1">
+              Choose a file to upload
+              <IconButton
+                onClick={() => {
+                  // @ts-ignore
+                  fileInput.current.click();
+                }}
+              >
+                <AttachIcon />
+              </IconButton>
+            </Typography>
+            <input
+              type="file"
+              name="upload"
+              accept=".pdf"
+              className={styles.fileInput}
+              ref={fileInput}
+              onChange={handleFileInputChange}
+            />
+            {!!pdfName ? (
```

Comment:
Why do we use separate state, we can use `formData.pdfFile?.name`

Reply:
Good point - I'll update!